modules/colors.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <title>Color picker</title>
+    <style>
+      dialog {
+        border: 1px solid;
+        border-radius: 5px;
+        box-shadow: 3px 3px 10px rgb(0 0 0 / 0.2);
+        background-color: white;
+        font-family: segue, arial, helvetica, sans-serif;
+        margin-top: 5vh;
+      }
+      table {
+        width: 100%;
+      }
+      td,
+      th {
+        padding: 3px 15px;
+      }
+      th {
+        background-color: #ededed;
+      }
+      td {
+        background-color: #dedede;
+        font-family: monospace;
+      }
+      body {
+        background: linear-gradient(
+            -90deg,
+            transparent 0 38%,
+            47%,
+            rgb(0 0 0 / 0.7) 50%,
+            white 50% 100%
+          ),
+          conic-gradient(
+            black 0 90deg,
+            transparent 90deg 180deg,
+            black 180deg 270deg,
+            transparent 270deg 360deg
+          );
+        background-size: 100% 100%, 20px 20px;
+        padding: 0;
+        margin: 0;
+      }
+      div {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+  <body>
+    <div>
+      <dialog>
+        <table>
+          <caption>
+            Current color values:
+          </caption>
+          <tbody>
+            <tr id="RGB">
+              <th>RGB</th>
+              <td></td>
+            </tr>
+            <tr id="HEX">
+              <th>HEX</th>
+              <td></td>
+            </tr>
+            <tr id="HSL">
+              <th>HSL</th>
+              <td></td>
+            </tr>
+            <tr id="HWB">
+              <th>HWB</th>
+              <td></td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p>
+          <label for="color">Select a color: </label
+          ><input type="color" id="color" value="#bada55" />
+        </p>
+        <p>
+          <label for="opacity">Select an opacity: </label
+          ><input
+            type="range"
+            id="opacity"
+            value="1"
+            min="0"
+            max="1"
+            step="0.1"
+          />
+        </p>
+      </dialog>
+    </div>
+    <script>
+      const colorPicker = document.getElementById("color");
+      const opacityPicker = document.getElementById("opacity");
+
+      document.addEventListener("readystatechange", () => {
+        document.querySelector("dialog").show();
+        createColor();
+      });
+
+      colorPicker.addEventListener("change", () => {
+        createColor();
+      });
+
+      opacityPicker.addEventListener("change", () => {
+        createColor();
+      });
+
+      function createColor() {
+        const currentColor = colorPicker.value;
+        const currentOpacity = opacityPicker.value * 1;
```

Comment:
See https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#type_coercion
```suggestion
        const currentOpacity = Number(opacityPicker.value);
```